content/try-opensds/installation-using-ansible.md

```diff
@@ -0,0 +1,278 @@
+---
+title: Installation Using ansible
+description: "This is a  guide for users and new contributors to install Local Cluster using ansible."
+weight: 30
+---
+This document describes how to install SODA projects local cluster, including Hotpot (API, Controller, Dock), Gelato, Telemetry, Orchestration, and Dashboard components.
+
+`Hereafter, Hotpot refers to API, Controller and Dock projects`
+
+### Pre-config (Ubuntu 16.04)
+All the installation work is tested on `Ubuntu 16.04`, please make sure you have installed the right one. Also `root` user is REQUIRED before the installation work starts.
+
+<br /> Install following packages:
+
+```bash
+apt-get update && apt-get install -y git make curl wget libltdl7 libseccomp2 libffi-dev gawk
+```
+<br />Install docker:
+
+```bash
+wget https://download.docker.com/linux/ubuntu/dists/xenial/pool/stable/amd64/docker-ce_18.06.1~ce~3-0~ubuntu_amd64.deb
+```
+```bash
+dpkg -i docker-ce_18.06.1~ce~3-0~ubuntu_amd64.deb
+```
+<br /> Install docker-compose:
+
+```bash
+curl -L "https://github.com/docker/compose/releases/download/1.23.1/docker-compose-$(uname -s)-$(uname -m)" -o /usr/local/bin/docker-compose
+chmod +x /usr/local/bin/docker-compose
+```
+<br /> Install golang
+
+```bash
+wget https://storage.googleapis.com/golang/go1.12.1.linux-amd64.tar.gz
```

Comment:
Golang version is going to be updated to v1.13 (exact version to be provided shortly)

Reply:
@skdwriting waiting for this